glossary.asciidoc

```diff
@@ -158,7 +156,6 @@ digital signature::
 double-spending::
     Double-spending is the result of successfully spending some money more than once.
     Bitcoin protects against double-spending by verifying that each transaction added to the blockchain plays by the rules; this means checking that the inputs for the transaction have not previously already been spent.
-    The Revocable Sequence Maturity Contracts used to construct payment channels heavily attempt to double-spend bitcoin.
```

Comment:
While above I tend to agree that removing the example might be good in this particular case I tend towards keeping it. The RSMC and doublespending is in my opinion a very crucial part of the protocol

Reply:
Ok, agreed.  (I'm learning about Lightning as I read, so it's difficult for me to judge the relative importance of information. )